src/transformers/tape.js

```diff
@@ -78,8 +78,6 @@ const tPropertiesUnsupported = new Set([
     'notDeepLooseEqual',
     'notLooseEqual',
     'notLooseEquals',
-
-    'skip',
```

Comment:
Hey @skovhus, I've added that test, but it doesn't fail for me. Shouldn't it fail so that we can ensure coverage for adding back in this removed line?